org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/IXMLCommandService.java

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Pivotal, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Pivotal, Inc. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.lemminx.services;
```

Comment:
Please move IXMLCommandService to the package `org.eclipse.lemminx.services.extensions.commands`.

My idea is to provide some abstract class for command like abstract command class which have an uri in the argument to get the DOMDocument from an uri.

Providing this kind of class will force to use for instance uri as parameter name for retrieving a DOMDocument.

Reply:
Fixed as suggested